app/browser/reducers/historyReducer.js

```diff
@@ -60,19 +55,15 @@ const historyReducer = (state, action, immutableAction) => {
         if (Immutable.List.isList(action.get('historyKey'))) {
           action.get('historyKey', Immutable.List()).forEach((key) => {
             state = historyState.removeSite(state, key)
+            // TODO: Remove history site Sync
```

Comment:
It's not a current Sync feature, so I think we should add it after the base Sync fixes are merged.